Lib/test/test_complex.py

```diff
@@ -814,7 +814,7 @@ def test_format(self):
         self.assertRaises(ValueError, (1.5+3j).__format__, '=20')
 
         # integer presentation types are an error
-        for t in 'bcdoxX':
+        for t in 'bcdo':
```

Comment:
It seems like you added support for "x" in complex numbers. Would it be possible to not add it?

Reply:
I think so, but why we want this?